tests/lib/rules/id-length.js

```diff
@@ -64,6 +64,8 @@ ruleTester.run("id-length", rule, {
         { code: "({ a: obj.x.y.z } = {});", options: [{ properties: "never" }], languageOptions: { ecmaVersion: 6 } },
         { code: "import something from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
         { code: "export var num = 0;", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
+        { code: "import * as something from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
+        { code: "import { something } from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
```

Comment:
```suggestion
        { code: "import { x } from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
```

It shows that `ImportSpecifiers` are allowed even with default `2`.

Reply:
Good point. 
I think we should add opposite case like
```javascript
{ code: "import { longName } from 'y';", options: [{ max: 5 }], languageOptions: { ecmaVersion: 6, sourceType: "module" } },```